src/lib/components/accordion/index.tsx

```diff
@@ -0,0 +1,147 @@
+
+  const handleClick = useCallback((questionData:  AccordionItem) => {
+    const { id } = questionData;
+    if (selectedQuestionId.includes(id)) {
+      if (!multiple) {
+        setSelectedQuestionId([]);
+        return
+      }
+
+      setSelectedQuestionId((selectedItems) => 
+        selectedItems.filter((selectedId) => selectedId !== id)
+      );
+    } else {
+      onClick?.(questionData);
+      setSelectedQuestionId((selectedIds) => [
+        id, ...multiple ? selectedIds : [],
+      ]);
+    }
+  }, [multiple, onClick, selectedQuestionId]);
+
+  return (
+    <div
+      className={
+        classnamesUtil(
+          styles.wrapper,
+          {
+            'br8': !isDefaultVariant,
+            [styles.wrapperBordered]: !isDefaultVariant,
+          }
+        )
+      }
+    >
+      {items.map((questionData) => {
+        const { question, answer, id, icon } = questionData;
+        const isOpen = selectedQuestionId.includes(id);
+
+        return (
+          <div
+            key={id}
+            className={classnamesUtil(
+              classNames?.questionWrapper,
+              styles.container,
+              { [styles.containerBordered]: !isDefaultVariant }
+            )}
+          >
+            <Card
+              title={question}
+              titleVariant="small"
+              density='compact'
+              icon={icon}
+              classNames={{
+                icon: classnamesUtil(
+                  classNames?.icon,
+                  'tc-grey-700'
+                ),
+                buttonWrapper: classnamesUtil(
+                  classNames?.buttonWrapper,
+                  styles.buttonWrapper,
+                  { 'py8': isDefaultVariant, },
+                ),
+                wrapper: classnamesUtil(
+                  classNames?.wrapper,
+                  'bg-transparent br0',
+                  { 'pl0': isDefaultVariant },
+                ),
+                title: classnamesUtil(
+                  classNames?.question,
+                  'tc-grey-700 fw-bold'
+                ),
+                ...classNames,
+              }}
+              dropShadow={false}
+              onClick={() => handleClick(questionData)}
+              aria-expanded={isOpen}
+              actionIcon={
+                <ChevronDownIcon
+                  className={classnamesUtil(styles.chevron, {
+                    [styles.chevronOpen]: isOpen
+                  })}
+                  color="grey-700"
+                  size={20}
+                  noMargin
+                />
+              }
+            />
+
+            <AnimateHeight
+              duration={300}
+              height={isOpen ? 'auto' : 0}
+            >
+              <div 
+                className={classnamesUtil(
+                  classNames?.answer,
+                  'tc-grey-600 p16 pl0', 
+                  { 'pl16': !isDefaultVariant },
+                )}
+              >
+                {answer}
```

Comment:
There is no Markdown integration, that needs to be done in other repositories as it differs between website/app.

Reply:
Can you elaborate?

Reply:
Sure! On both [app](https://github.com/getPopsure/app/blob/main/src/components/markdown/index.tsx) and [website](https://github.com/getPopsure/website/blob/main/components/markdown/index.tsx) we have different custom markdown integrations so we'll need to configure components to have their own markdown on answers.

I'm talking about this part that wasn't implemented on design system in this PR:
https://github.com/getPopsure/app/blob/main/src/components/faq/index.tsx#L77